src/styles/FormInput.css

```diff
@@ -27,12 +27,20 @@
   letter-spacing: 3px;
 }
 
-.input-shown {
-  width: 100%;
-  max-width: 270px;
-  height: 50px;
-  padding: 15px;
+textarea {
+  overflow: hidden;
+  padding: 10px;
+  width: 250px;
```

Comment:
Maybe you can find a way to give he textarea a relative width using percentages for instance? (instead of a fixed one). That way would be more adapted to screen size changes.
Just an idea, not critical